static/js/shared/child_places.tsx

```diff
@@ -42,16 +45,23 @@ export function ChildPlaces(props: ChildPlacesPropType): JSX.Element {
           <div className="child-place-type">
             {displayNameForPlaceType(placeType, true /* isPlural */)}
           </div>
-          {props.childPlaces[placeType].map((place, i) => (
-            <a
-              href={"/disasters/" + place.dcid}
-              className="child-place-link"
-              key={`child-place-${i}`}
-            >
-              {place.name || place.dcid}
-              {i < props.childPlaces[placeType].length - 1 ? "," : ""}
-            </a>
-          ))}
+          {props.childPlaces[placeType].map((place, i) => {
+            const rs: ReplacementStrings = {
+              place: "",
```

Comment:
maybe we should update place and date to also be optional since there seems to be a lot of cases where it's just empty string passed for those fields? Can just add a TODO

Reply:
i'll add a TODO and update in a followup